docs/readthedocs/source/doc/Chronos/Overview/install.md

```diff
@@ -19,7 +19,9 @@
 .. note::
     **Supported Python Version**:
 
-     Chronos only supports Python 3.7.2 ~ latest 3.7.x. We are validating more Python versions.
+     Chronos supports all installation options on Python 3.7.2 ~ latest 3.7.x. For details about different installation options, refer to `here <#install-using-conda>`_.
+
+     Moreover, ``bigdl-chronos[pytorch]``, ``bigdl-chronos[tensorflow]``, ``bigdl-chronos[automl]`` and ``bigdl-chronos[inference]`` are supported on Python 3.8.
```

Comment:
all version of python 3.8 works？ or just a few versions like 3.8.x

Reply:
> all version of python 3.8 works？ or just a few versions like 3.8.x

The latest python 3.8 works.

Reply:
I think if the latest 3.8 works (and we have established a CI/CD job for it). We could claim that 3.8 is supported.